index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Covid Guidlines | Mansory Layout</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main class="container">
+      <div class="box">
+        <img src="washing-hand.png" alt="" />
+        <p>
+          Wash hands with soap and water after touching animals and animal
+          products.
+        </p>
+      </div>
+      <div class="box">
+        <img src="cold.png" alt="" />
+        <p>
+          When coughing and sneezing cover mouth and nose with flexed elbow or
+          tissue.
+        </p>
+      </div>
+      <div class="box">
+        <img src="flight.png" alt="" />
```

Comment:
```alt``` attribute will tell user about the image of the browsers fail to load it. If you will leave it blank then user might not understand what the image was about.